packages/db/prisma/prismaClient.ts

```diff
@@ -1,19 +1,23 @@
-import { PrismaClient } from "@prisma/client";
+import { PrismaClient } from "./generated/client";
+import { PrismaPg } from "@prisma/adapter-pg";
 import { readReplicas } from "@prisma/extension-read-replicas";
 
+const adapter = new PrismaPg({ connectionString: process.env.DATABASE_URL });
```

Comment:
interesting, PrismaPg does indeed have slightly more flexibility, what was your reason for?